modules/localgov_geo_address/templates/localgov-geo--address--embed.html.twig

```diff
@@ -14,8 +14,13 @@
  * @see template_preprocess_localgov_geo()
  */
 #}
+
 <article{{ attributes }}>
   {% if content %}
-    {{- content|without('label') -}}
+    {% if content.location['#object'].location|length == 0 %}
```

Comment:
`if content.location['#object'].location is empty` is returning as `false` whether we have lat/long filled in or not. The check I do returns `1` if lat/long is filled in and `0` if not.

Reply:
God I hate twig.

Reply:
OK no I hate twig *sometimes*.

I think this is probably because it's a fieldlist. So querying the iterator says there's no value in the list. Better would probably be to do this in the preprocess and add some variable. But it's probably not worth it as in the default a geo value is compulsory for a geo entity anyway. It certainly wouldn't make sense for the area version.